.github/CONTRIBUTING.md

```diff
@@ -7,32 +7,31 @@ Primarily, Armory has two main modes of operation:
   - Native (also known as `--no-docker`) mode - This uses a pre-set python environment to execute the configuration file.
```

Comment:
You convinced me by example, .github is a fine place for that doc (but not the README.md ;)